editor/src/diagram/di.config.ts

```diff
@@ -96,6 +97,7 @@ const ivyDiagramModule = new ContainerModule((bind, unbind, isBound, rebind) =>
 
   configureIvyModelElement(EventIntermediateTypes.INTERMEDIATE_TASK, EventNode, IntermediateEventNodeView);
   configureIvyModelElement(EventIntermediateTypes.INTERMEDIATE_WAIT, EventNode, IntermediateEventNodeView, { enable: [goToSourceFeature] });
+  configureIvyModelElement(EventIntermediateTypes.INTERMEDIATE_THIRD_PARTY, EventNode, IntermediateEventNodeView, { enable: [goToSourceFeature] });
```

Comment:
does this support the "gotosource" feature?

Reply:
actually not ... gonna remove it :)